.github/workflows/Release-Please.yml

```diff
@@ -29,8 +29,8 @@ jobs:
           curl -X POST \
           -H "Accept: application/vnd.github.v3+json" \
           -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
-          https://api.github.com/repos/spotify/confidence-sdk-js/dispatches \
-          -d '{"event_type":"release_amend_trigger", "client_payload": {}}'
+          https://api.github.com/repos/spotify/confidence-sdk-js/actions/workflows/Amend-Release.yml/dispatches \
+          -d '{"ref":"release-please--branches--main", "inputs": {"branch": "release-please--branches--main"}}'
```

Comment:
maybe ref should be set to `main`?
AFAIK this is which "version" of the workflow should be run and I assume we want the one from main to run?